pytket/pytket/qasm/qasm.py

```diff
@@ -160,6 +160,7 @@ class QASMUnsupportedError(Exception):
     "bridge": OpType.BRIDGE,
     "iswapmax": OpType.ISWAPMax,
     "zzmax": OpType.ZZMax,
+    "ecr": OpType.ECR,
```

Comment:
The last commit from Silas is not updating the changelog? Should that be added?

Reply:
Done.